### DISCUSSION TITLE
Adding Ground State Identifier as Feature to the Model

### DISCUSSION THREAD

**Discussion by rydeveraumn** [materialyzeai/matgl#199] [General]
Hello all,

I am new to materials science and am working through a project where we would like to continue to build off of the ideas that have been presented in this repository. One of the ideas 💡 that have been presented is using the ground state information (binary attribute) of whether a structure is ground state or not. Would this be represented as a global level features for each individual graph? Has this information been used before?

**Comment by shyuep** [comment]
Yes, the global state embedding would be the way to do this.
